### DISCUSSION TITLE
How to set min number of nodes in a single row in Directed Layout Graph.

### DISCUSSION THREAD

**Discussion by jibinjoy8897** [clientIO/joint#2557] [Q&A]
### Introduction

I have a autoresize Paper. I have to have a fixed number(suppose 5 nodes in Ist row and the 6th node starts in the next row of directed graph) of nodes in each row using directedGraph.Please look at my current code

`layout.DirectedGraph.layout(graph, {
      graphlib: dagre.graphlib,
      dagre: dagre,
      setVertices: true,
      rankDir: 'TB',
      marginX: xOffset,
      marginY: yOffset,
      nodeSep: colGap,
      rankSep: rowGap,
    });`

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
There is no such an option in [dagre](https://github.com/dagrejs/dagre) library. For more info please contact devs at `dagre`.

_Note that when it comes to automatic layout, you need to be much more specific about your requirements. e.g. based on your description it's not clear what should happen with children of the 1st element in the row (because they could overlap with the 5th element)._
